api/v1alpha1/edgeworkload_types.go

```diff
@@ -33,13 +33,20 @@ type EdgeWorkloadSpec struct {
 	Data            *DataConfiguration             `json:"data,omitempty"`
 	ImageRegistries *ImageRegistriesConfiguration  `json:"imageRegistries,omitempty"`
 	Metrics         *ContainerMetricsConfiguration `json:"metrics,omitempty"`
+	SecurityContext *EdgeWorkloadSecurityContext   `json:"securityContext,omitempty"`
 
 	// LogCollection is the logCollection property to be used to collect logs
 	// from this endpoint. This key is what is defined on the edgedevice
 	// logCollection property
 	LogCollection string `json:"logCollection,omitempty"`
 }
 
+// EdgeWorkloadSecurityContext holds workload-level security attributes
+type EdgeWorkloadSecurityContext struct {
+	// RunByRoot determines if the containers in the workload should run under the root user. By default
+	// workloads are run by a non-root user.
+	RunByRoot *bool `json:"runByRoot,omitempty"`
```

Comment:
Is the demand only to be run as root or as flotta user or will there be a case for "RunAs" and provide the user that should execute this workload?

Reply:
As far as I know either as root or as the flotta user.